MastodonSDK/Sources/MastodonUI/View/Content/StatusMetricRowView.swift

```diff
@@ -0,0 +1,119 @@
+// Copyright © 2023 Mastodon gGmbH. All rights reserved.
+
+import UIKit
+import MastodonAsset
+
+public final class StatusMetricRowView: UIButton {
+    let icon: UIImageView
+    let textLabel: UILabel
+    let detailLabel: UILabel
+    let chevron: UIImageView
+
+    private let contentStack: UIStackView
+
+    public init(iconImage: UIImage? = nil, text: String? = nil, detailText: String? = nil) {
+
+        icon = UIImageView(image: iconImage?.withRenderingMode(.alwaysTemplate))
+        icon.tintColor = Asset.Colors.Label.secondary.color
+        icon.translatesAutoresizingMaskIntoConstraints = false
+
+        textLabel = UILabel()
+        textLabel.translatesAutoresizingMaskIntoConstraints = false
+        textLabel.font = UIFontMetrics(forTextStyle: .body).scaledFont(for: .systemFont(ofSize: 17, weight: .regular))
+        textLabel.textColor = Asset.Colors.Label.primary.color
+        textLabel.numberOfLines = 0
+        textLabel.text = text
+
+        detailLabel = UILabel()
+        detailLabel.translatesAutoresizingMaskIntoConstraints = false
+        detailLabel.text = detailText
+        detailLabel.textColor = Asset.Colors.Label.secondary.color
+        detailLabel.numberOfLines = 0
+        detailLabel.font = UIFontMetrics(forTextStyle: .body).scaledFont(for: .systemFont(ofSize: 17, weight: .regular))
+
+        chevron = UIImageView(image: UIImage(systemName: "chevron.right"))
+        chevron.translatesAutoresizingMaskIntoConstraints = false
+        chevron.tintColor = Asset.Colors.Label.tertiary.color
+
+        contentStack = UIStackView()
+        contentStack.translatesAutoresizingMaskIntoConstraints = false
+        contentStack.distribution = .fill
+        contentStack.spacing = 8
+        contentStack.isUserInteractionEnabled = false
+        contentStack.addArrangedSubview(textLabel)
+        contentStack.addArrangedSubview(detailLabel)
+
+        super.init(frame: .zero)
+
+        self.traitCollectionDidChange(nil)
+
+        addSubview(icon)
+        addSubview(contentStack)
+        addSubview(chevron)
+
+        setupConstraints()
+    }
+
+    required init?(coder: NSCoder) { fatalError("init(coder:) has not been implemented") }
+
+    public override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+
+        if traitCollection.preferredContentSizeCategory.isAccessibilityCategory {
+            contentStack.axis = .vertical
+            contentStack.alignment = .fill
+            detailLabel.textAlignment = .natural
+        } else {
+            contentStack.axis = .horizontal
+            contentStack.alignment = .leading
```

Comment:
```suggestion
            contentStack.alignment = .center
```

![stackview-center](https://user-images.githubusercontent.com/2580019/221036642-86dcb206-def8-4c47-b1b3-809a4d957250.jpeg)

Reply:
Will add that, working on a couple of fixes here too! (the detail should be on  line)

Reply:
Don't worry, it's basically already taken care of with this nice suggestion-feature :)

Reply:
Done in #953, also fixed the issue where the text would break onto 2 lines even if there was enough space for it to be on 1

Reply:
Thank you @j-f1 ! 👏